docs/competency-api/competency.md

```diff
@@ -4,4 +4,63 @@ title: Competency
 nav_order: 1
 parent: Competency API
 ---
-# Competency
\ No newline at end of file
+# Competency
+
+## Create Competency
+
+### Description
+This route is used to create a competency.
+
+#### Post: 
+```http
+https://api.competency-constructor.clark.center/competencies
```

Comment:
can you add curl requests for both of these routes? it would look like the other examples but just remove the line where the body is sent, update the url, and the http method (POST/PATCH/DELETE... etc)

Reply:
sure, the curl requests are in now